content/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/endpoint.md

```diff
@@ -0,0 +1,78 @@
+---
+title: webexteams.endpoint() function
+description: >
+  The `webexteams.endpoint()` function sends a single message to Webex using the
+  [Webex messages API](https://developer.webex.com/docs/api/v1/messages/create-a-message).
+menu:
+  influxdb_2_0_ref:
+    name: webexteams.endpoint
+    parent: Webex Teams
+weight: 202
+---
+
+The `webexteams.endpoint()` function returns a function that sends a message that
+includes data from input rows to a Webex room.
+
+```js
+import "contrib/sranka/webexteams"
+
+webexteams.endpoint(
+  url: "https://webexapis.com",
+  token: "token"
+)
+```
+
+## Parameters
+
+### url {data-type="string"}
+Base URL of Webex API endpoint _(without a trailing slash)_.
+Default is `https://webexapis.com`.
+
+_**Data type**: String_
+
+### token {data-type="string"}
+({{< req >}})
+[Webex API access token](https://developer.webex.com/docs/api/getting-started).
+
+_**Data type**: String_
+
+## Usage
+`webexteams.endpoint` is a factory function that outputs another function.
+The output function requires a `mapFn` parameter.
+
+### mapFn
+A function that builds the object used to generate the POST request.
+Requires an `r` parameter.
+
+_**Data type:** Function_
+
+`mapFn` accepts a table row (`r`) and returns an object that must include the
+following fields:
+
+- `roomId`
+- `text`
+- `markdown`
+
+_For more information, see [`webexteams.message()` parameters](/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/message/#parameters)._
+
+## Examples
+
+##### Send the last reported status to Webex Teams
+```js
+import "contrib/sranka/webexteams"
+import "influxdata/influxdb/secrets"
+
+// this value can be stored in the secret-store()
```

Comment:
I prefer JSDOC (block) comment syntax. I know it's more verbose, but it's a visual indicator that I should read it, not ignore it as commented code.

```suggestion
/**
 * Use the secret-store() to store third-party tokens.
 */
```

Reply:
Yeah, Flux doesn't support block comments: https://docs.influxdata.com/influxdb/v2.0/reference/flux/language/lexical-elements/#comments.

I meant to remove this line altogether anyway. 
```suggestion
```